Makefile.arm7

```diff
@@ -63,7 +63,7 @@ endif
 # Source files
 # ------------
 
-SOURCES_S	:= $(shell find -L $(SOURCEDIRS) -name "*.s")
+SOURCES_S	:= $(shell find -L $(SOURCEDIRS) -name "*.S")
```

Comment:
This is a bad idea. Relying on the _case_ of the extension is going to cause issues for sure. We use lowercase extensions, but they should be considered case-insensitive.

I know that's how GCC does things, but that's a terrible idea, and it's not part of the C standard, as far as I know, so we should ignore that convention.

Reply:
Also, if you have a repository on Windows and the _case_ of a file name changes, the actual file name won't change because Windows thinks they are the same name, so this isn't even going to work everywhere.

Reply:
If I'm not mistaken, searching for `.S` in a case-insensitive filesystem/OS should include `.s` files, so it should be fine on Windows. Now, it could cause issues if a `.s` file is made from windows and then compiled on Linux, so I think I'll just revert those changes keeping with the current lowercase convention.